packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -446,85 +447,103 @@ const AccountSwitcher = ({
+            {is_landing_company_loaded ? (
+                <React.Fragment>
+                    <Tabs
+                        active_index={active_tab_index}
+                        className='acc-switcher__list-tabs'
+                        onTabItemClick={index => setActiveTabIndex(index)}
+                        top
+                    >
+                        {/* TODO: De-couple and refactor demo and real accounts groups
                         into a single reusable AccountListItem component */}
-                <div label={localize('Real')} id='real_account_tab'>
-                    <DesktopWrapper>
-                        <ThemedScrollbars height='354px'>{real_accounts}</ThemedScrollbars>
-                    </DesktopWrapper>
-                    <MobileWrapper>
-                        <Div100vhContainer className='acc-switcher__list-container' max_autoheight_offset='234px'>
-                            {real_accounts}
-                        </Div100vhContainer>
-                    </MobileWrapper>
-                </div>
-                <div label={localize('Demo')} id='dt_core_account-switcher_demo-tab'>
-                    <DesktopWrapper>
-                        <ThemedScrollbars height='354px'>{demo_account}</ThemedScrollbars>
-                    </DesktopWrapper>
-                    <MobileWrapper>
-                        <Div100vhContainer className='acc-switcher__list-container' max_autoheight_offset='234px'>
-                            {demo_account}
-                        </Div100vhContainer>
-                    </MobileWrapper>
-                </div>
-            </Tabs>
-            <div
-                className={classNames('acc-switcher__separator', {
-                    'acc-switcher__separator--auto-margin': is_mobile,
-                })}
-            />
-            <div className='acc-switcher__total'>
-                <Text line_height='s' size='xs' weight='bold' color='prominent'>
-                    <Localize i18n_default_text='Total assets' />
-                </Text>
-                <Text size='xs' color='prominent' className='acc-switcher__balance'>
-                    <Money
-                        currency={isRealAccountTab ? account_total_balance_currency : vrtc_currency}
-                        amount={formatMoney(
-                            isRealAccountTab ? account_total_balance_currency : vrtc_currency,
-                            isRealAccountTab ? getTotalRealAssets() : getTotalDemoAssets(),
-                            true
-                        )}
-                        show_currency
-                        should_format={false}
+                        <div label={localize('Real')} id='real_account_tab'>
+                            <DesktopWrapper>
+                                <ThemedScrollbars height='354px'>
+                                    {is_landing_company_loaded ? real_accounts : <Loading is_fullscreen={false} />}
```

Comment:
I think this check can be removed now since there is a check for `is_landing_company_loaded` on the root level?